pages/implementation/reference/javascript.mdx

```diff
@@ -75,92 +68,100 @@ Avo.signupStart({ referral: 'direct' });
 
->**Arguments**
->
->`systemProperties`: an object where each field represents a system propety in your tracking plan. 
->When you define system properties in your Avo workspace you set name and type - the fileds of this object are named the same as system propertyies, in lower camel case, and you should provide corresponding types.
+##### Arguments
+
+`systemProperties`: an object where each field represents a system property in your tracking plan. 
+When you define system properties in your Avo workspace you set name and type - the fields of this object are named the same as system properties, in camelCase, and you should provide corresponding types.
 
-##### Event tracking functions
+#### Event tracking functions
 
 ```javascript
-Avo.yourEventName(properties: { [anonymousUserId_], [userId_], [segmentContext_], [eventProperty0], [eventProperty1], ..., [userProperty0], [userProperty1], ... })
+Avo.yourEventName(properties: { [eventProperty0], [eventProperty1], ..., [userProperty0], [userProperty1], ..., [userId_], [anonymousUserId_], [segmentContext_], })
 ```
 
-Every event you define in your tracking plan in Avo gets a function named after the event in lower camel case. The arguments of the function depend on how it's defined in your tracking plan 
-
-> **Arguments**
->
-> `anonymousUserId_`: string, Node.js only, this argument is automatically added if corresponding setting is enabled, user to track anonymous users
->
-> `userId_`: string
-> Node.js: added to all events, you have to either provide it or the `anonymousUserId_`
-> Web and React Native: Added if the event has the Identify User action
->
-> `eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any
-> This is an event property that should be tracked with given event. Pass the value of the property to track here.
->
-> `userProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any
-> This is a user property that should be updated with given event. Pass the value of the property to update here.
->
-> `segmentContext_`: object, Node.js only, passed down to Segment as the Segment constext, e.g. `segment.track({..., context: context})`
+Every event you define in your tracking plan in Avo gets a function named after the event in camelCase. The arguments of the function depend on how it's defined in your tracking plan 
+
+##### Arguments
+
+`eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any
+This is an event property that should be tracked with given event. Pass the value of the property to track here.
+
+`userProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any
+This is a user property that should be updated with given event. Pass the value of the property to update here.
```

Comment:
Here we should highlight that the event and property keys are camelCased versions of the event and property names defined in Avo